packages/protonup-qt-git/protonup-qt-git.pacscript

```diff
@@ -0,0 +1,36 @@
+name="protonup-qt-git"
+description="Install and manage Proton-GE and Luxtorpeda for Steam and Wine-GE for Lutris"
+url="https://github.com/DavidoTek/ProtonUp-Qt.git"
+
+depends="python3"
+build_depends="python3 python3-pip"
+
+pkgver() {
+    # get the latest tag (newest release version) - master may be broken, tags are releases
+    git ls-remote --tags "${url}" | tail -n1 | cut -f1 | cut -c1-8
+}
+version="$(pkgver)"
+
+prepare() {
+    sudo mkdir -p "${STOWDIR}/${name}/usr/bin"
+
+    sed 's/==.*//g' requirements.txt > requirements_nover.txt
+    pip3 install --verbose --no-index --ignore-installed --prefix="${STOWDIR}/${name}/usr" -r requirements_nover.txt
+
+cat >"${STOWDIR}/${name}/usr/bin/net.davidotek.pupgui2.sh" <<EOL
+#!/bin/bash
+PYTHONPATH=$PYTHONPATH:${STOWDIR}/${name}/usr/local/lib/`ls ${STOWDIR}/${name}/usr/local/lib/ | grep python3`/dist-packages/ python3 -m pupgui2
+EOL
+
+    sudo ln -s "${STOWDIR}/${name}/usr/bin/net.davidotek.pupgui2.sh" "/usr/bin/protonup-qt"
```

Comment:
This won't work. You probably want `sudo ln -s /usr/bin/net.davidotek.pupgui2.sh /usr/bin/protonup-qt` in `postinst`